Assets/Prefabs/EasyMove.cs

```diff
@@ -17,20 +18,22 @@ public class EasyMove : MonoBehaviour {
 	// Update is called once per frame
 	void Update () {
 
-		if (Input.GetKey (KeyCode.A))
+		if (Input.GetKey (KeyCode.A)) 
 			movex = -1;
 		else if (Input.GetKey (KeyCode.D))
 			movex = 1;
 		else
 			movex = 0;
-		if (Input.GetKey (KeyCode.W)) {
+		if (Input.GetKey (KeyCode.W) && moveYDis < 6) {
```

Comment:
I would like if we could do this without using a 'magic number'

Is there a way to derive this number from the GameObject that the character is constrained to?